README.md

```diff
@@ -17,22 +18,28 @@ export enum EventDirection {
 
 ```
 
-**queueUrl**(required) SQS queue url
+**queueLink**(required) Prefix to get the queue details from environment variable, for example
+if queueLink = 'CITIZEN', queue details are read from CITIZEN_URL, CITIZEN_NAME, CITIZEN_ARN
 
-**direction**(required) indicates whenther to backup or restore the messages
+**queueUrl**(optional, if queueLink is passed) SQS queue url
+
+**direction**(required) indicates whether to backup or restore the messages
 
 **path**(required) s3 file path relative to `APPDATA_BUCKET/APPDATA_PREFIX`
 
 **delete**(optional) indicates whether to delete the messages from the queue after the messages are backed up in s3 (default: false)
 
 ## Environment Variables
 
-| variable       | description                              |
-| -------------- | ---------------------------------------- |
-| LOG_LEVEL      | log level for pino, defaults to debug    |
-| AWS_REGION     | AWS region, defaults to ap-southeast-2   |
-| APPDATA_BUCKET | S3 bucket for storing backed-up messages |
-| APPDATA_PREFIX | S3 prefix for storing backed-up messages |
+| variable          | description                              |
+| ----------------- | ---------------------------------------- |
+| LOG_LEVEL         | log level for pino, defaults to debug    |
+| AWS_REGION        | AWS region, defaults to ap-southeast-2   |
```

Comment:
Still needed? maybe optional is url used

Reply:
still need it for s3, otherwise uses Us regions as default